cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/ContentRepository.kt

```diff
@@ -13,4 +13,9 @@ class ContentRepository(private val path: String = "/data/content") {
         val file = File("$path/$widgetId.json")
         return if (file.exists()) JsonObject(file.readText()) else JsonObject()
     }
+
+    fun delete(widgetId: String) {
+        val file = File("$path/$widgetId.json")
```

Comment:
you could probably use let here:
`File("$path/$widgetId.json").exists().let { it.delete() }`

Reply:
Unfortunately not, `it` would be a boolean, not a file, and you'd need to check for its value anyway.

Reply:
true